tf-modules/cumulus_distribution/iam.tf

```diff
@@ -46,6 +46,17 @@ data "aws_iam_policy_document" "lambda_distribution_api_gateway_policy" {
     resources = [aws_dynamodb_table.access_tokens.arn]
   }
 
+  statement {
+    actions = [
+      "s3:GetObject*",
```

Comment:
It seems we only need `"s3:GetObject*"` action based on my testing.

Reply:
Ah yeah, that makes sense. Removed everything else